createdisk-library.sh

```diff
@@ -86,33 +86,19 @@ EOF
 
 function create_qemu_image {
     local destDir=$1
-    local base=$2
-    local overlay=$3
+    local tempFile="temp_${SNC_PRODUCT_NAME}"
 
-    if [ -f /var/lib/libvirt/images/${base} ]; then
-      sudo cp /var/lib/libvirt/images/${base} ${destDir}
-    else
-      sudo cp /var/lib/libvirt/${SNC_PRODUCT_NAME}/${base} ${destDir}
-    fi
+    sudo cp /var/lib/libvirt/${SNC_PRODUCT_NAME}/${SNC_PRODUCT_NAME}.qcow2 ${destDir}
```

Comment:
Makes more sense to me to tell `sparsify` to create the final file, rather than creating to a temp file we rename right away.
```
sudo cp /var/lib/libvirt/${SNC_PRODUCT_NAME}/${SNC_PRODUCT_NAME}.qcow2 ${destDir}/${tempFile}
sparsify ${destDir}  ${tempFile} ${SNC_PRODUCT_NAME}.qcow2
```
Working directly from `/var/lib/libvirt` would even avoid the copy, but that's trickier as this requires changes in `sparsify`, there will be permission problems, ... so not required right now.